main_menu.rb

```diff
@@ -0,0 +1,45 @@
+# Class responsible for handling user input and displaying menu options
+class MainMenu
+  OPTIONS = {
+    '1' => :list_all_books,
+    '2' => :list_all_people,
+    '3' => :create_person,
+    '4' => :create_book,
+    '5' => :create_rental,
+    '6' => :list_rentals,
+    '7' => :exit
+  }.freeze
+
+  def initialize(app)
+    @app = app
+  end
+
+  # rubocop:disable Metrics/MethodLength
```

Comment:
- [ ] @Zeph180 and @djo1975 it's not good practice disabling the linter check in your code, I would advise you to try and fix the error, disabling the linter should be the last resort after trying and seeking help without success.

- [ ] @team I noted you still have the input and input validation in your app class  as shown by one example below, all inputs and input validation code need to be in the same class in order for you to achieve the `Single responsibility rule`, kindly ensure all requests for input and input validation should've been placed in the main.rb file and/or classes/methods whose responsibilities are to request or validate the input(main_menu.rb).
![image](https://user-images.githubusercontent.com/61408860/233975503-e69d68ad-b87f-4b3d-b193-8b7dc9cba4df.png)

Reply:
@danmainah 
Thank you for this feedback.
However, it does not provide the entire issues to be fixed considering the fact that you said "One example below" Could you please throw more light on the other issues we are supposed to fix

Reply:
- [ ] @Zeph180 any code that requires for user input and validation should be moved from the app.rb file.

Reply:
**At this stage we have to re-factorize main.rb**
Project requirements

At this point your codebase is pretty big. The main.rb file may be especially large at this point. Let's try to clean things up!

    Make sure YAGNI, KISS, and DRY rules and Ruby style guide are respected.
    Follow SOLID principles.
    Pay special attention to the "Single-responsibility Principle".
        Make sure that the role of main.rb is only to start your code and ask the user for the initial choice of options (or to quit).
        Extract some pieces of logic into separate class(es).